spacy/lang/fr/stop_words.py

```diff
@@ -36,45 +37,48 @@
 l' l’ la laisser laquelle le lequel les lesquelles lesquels leur leurs longtemps
 lors lorsque lui lui-meme lui-même là lès
 
-m' m’ ma maint maintenant mais malgre me meme memes merci mes mien
+m' m’ ma maint maintenant mais malgre malgré me meme memes merci mes mien
 mienne miennes miens mille moi moi-meme moi-même moindres moins
 mon même mêmes
 
 n' n’ na ne neanmoins neuvième ni nombreuses nombreux nos notamment
-notre nous nous-mêmes nouvea nul néanmoins nôtre nôtres
+notre nous nous-mêmes nouveau nul néanmoins nôtre nôtres
 
-o ô on ont onze onzième ore ou ouias oust outre
+o ô on ont onze onzième or ou ouias ouste outre
```

Comment:
- changed "ore" (rarely used word in French) to "or" (a common conjonction de coordination that should be included, like "mais", "donc", etc.);
- changed "oust" to "ouste" (correct interjection);